src/views/design/ScriptsOverview/index.tsx

```diff
@@ -333,28 +339,50 @@ const ScriptsOverview = withStyles(styles)(
                                                 },
                                             });
                                         },
-                                    }, {
-                                        icon: <ReportIcon />,
-                                        label: translator('scripts.overview.list.actions.report'),
-                                        onClick: (id) => {
-                                            const scripts = getAsyncScripts(this.props.state);
-                                            const selectedScript = scripts.find((item) =>
-                                                getUniqueIdFromScript(item) === id);
-
-                                            redirectTo({
-                                                routeKey: ROUTE_KEYS.R_REPORTS,
-                                                queryParams: {
-                                                    script: selectedScript.name,
-                                                    version: selectedScript.version.number,
-                                                },
-                                            });
-                                        },
-                                    }, {
-                                        icon: <Delete />,
-                                        label: translator('scripts.overview.list.actions.delete'),
-                                        onClick: this.setScriptToDelete,
-                                    },
-                                ]}
+                                    } : [],
+                                    !checkAuthority(SECURITY_PRIVILEGES.S_SCRIPTS_WRITE, 'PUBLIC')
+                                        && checkAuthority(SECURITY_PRIVILEGES.S_SCRIPTS_READ, 'PUBLIC') ? {
```

Comment:
Can we already use the security group name of the script in question instead of the fixed value 'PUBLIC'.